Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM golang:1.3
+FROM golang:1.4
 MAINTAINER Brian Tiger Chow <btc@perfmode.com>
 
 ADD . /go/src/github.com/jbenet/go-ipfs
 RUN cd /go/src/github.com/jbenet/go-ipfs/cmd/ipfs && go install
 
-EXPOSE 4001 5001 4002/udp
+RUN echo -n "#!/bin/bash\nipfs init\nipfs config Addresses.API /ip4/0.0.0.0/tcp/5001\nipfs config Addresses.Gateway /ip4/0.0.0.0/tcp/8080\nipfs daemon" > /usr/local/bin/start_ipfs && \
```

Comment:
maybe we could put this into a script in `bin/` (maybe `bin/container_daemon` or something) for cleaner readability.